Tools/TestWebKitAPI/Tests/WebCore/DocumentOrder.cpp

```diff
@@ -52,9 +53,7 @@ using namespace WebCore;
 
 static Ref<Document> createDocument()
 {
-    HTMLNames::init();
-    WebKitFontFamilyNames::init();
-    initializeCommonAtomStrings();
+    ProcessWarming::initializeNames();
```

Comment:
This probably doesn't belong.

Reply:
No, this is the fix for https://bugs.webkit.org/show_bug.cgi?id=244089.
The crash is caused by XMLName::langAttr being used inside Element::insertedInto
as this test directly calls into WebCore functions.